### ISSUE TITLE
Configure raises an error if CTest scripts print to stdout

### ISSUE THREAD

**Issue by benschreiber** [microsoft/vscode-cmake-tools#3750]
### Brief Issue Summary

My project uses [TEST_INCLUDE_FILES](https://cmake.org/cmake/help/latest/prop_dir/TEST_INCLUDE_FILES.html) to generate labels for CTest tests at runtime.

Our script prints a line `Generating test labels...` to stdout. This causes CMake Tools to report an error when configuring.

```
[rollbar] Unhandled exception: Unhandled Promise rejection: cleanConfigure SyntaxError: Unexpected token G in JSON at position 0 {}

[error] SyntaxError: Unexpected token G in JSON at position 0
	at JSON.parse (<anonymous>)
	at CTestDriver.refreshTests (/root/.vscode-server/extensions/ms-vscode.cmake-tools-1.17.17/dist/main.js:50861:37)
	at async /root/.vscode-server/extensions/ms-vscode.cmake-tools-1.17.17/dist/main.js:44894:29
	at async CMakeProject.doConfigure (/root/.vscode-server/extensions/ms-vscode.cmake-tools-1.17.17/dist/main.js:45001:24) cmake.cleanConfigure {"value":"ms-vscode.cmake-tools","_lower":"ms-vscode.cmake-tools"}
```

This can easily be reproduced.
```cmake
cmake_minimum_required(VERSION 3.27)
project(foo)

enable_testing()

file(CONFIGURE
    OUTPUT ${PROJECT_BINARY_DIR}/createTestLabels.cmake
    CONTENT "execute_process(COMMAND @CMAKE_COMMAND@ -E echo \"Generating test labels...\")\n"
    @ONLY
)

set_property(DIRECTORY APPEND PROPERTY
    TEST_INCLUDE_FILES ${PROJECT_BINARY_DIR}/createTestLabels.cmake
)
```
```json
{
    "version": 6,
    "configurePresets": [
        {
            "name": "gcc-debug",
            "binaryDir": "${sourceDir}/out/build/${presetName}",
            "cacheVariables": {
                "CMAKE_CXX_COMPILER": "g++",
                "CMAKE_BUILD_TYPE": "Debug"
            },
            "generator": "Unix Makefiles"
        }
    ],
    "buildPresets": [
        {
            "name": "gcc-debug",
            "configurePreset": "gcc-debug"
        }
    ],
    "testPresets": [
        {
            "name": "gcc-debug",
            "configurePreset": "gcc-debug"
        }
    ]
}
```

**Comment by Yingzi1234** [comment]
@benschreiber Thank you for your github issue! We followed the command you provided to reproduce the issue, but we encountered error below, in order to better reproduce and solve your issue, can you provide us with your reproduction steps video and reproduction file?
![image](https://github.com/microsoft/vscode-cmake-tools/assets/101234795/13125239-7ee3-418b-8d41-5f40f05b0303)

**Comment by benschreiber** [comment]
It looks like you are on windows? Then you can modify the preset to use MSVC and the VS Generator. But it would be easier to just try to reproduce on Linux or in WSL.

**Comment by benschreiber** [comment]
@Yingzi1234 If you're still stuck, try using this `devcontainer.json`
```json
{
    "image": "mcr.microsoft.com/devcontainers/cpp:debian-12",
    "customizations": {
        "vscode": {
            "extensions": [
                "ms-vscode.cpptools",
                "ms-vscode.cmake-tools"
            ]
        }
    }
}
```

**Comment by Yingzi1234** [comment]
@benschreiber Thanks for your reply, based on the latest information you provided, we tried to reproduce your issue on a Linux machine, but are not sure if we got the same results as your issue. When I run the "CMake: configure" command, an error box pops up, but when I go to the output window, the contents show that my configuration was successful. Here's a recording of my reproduction for you to check out.
[Uploading GitHub issue.webm…]()
